depends/packages/openssl.mk

```diff
@@ -7,28 +7,72 @@ $(package)_sha256_hash=a45de072bf9be4dea437230aaf036000f0e68c6a665931c57e76b5b03
 define $(package)_set_vars
 $(package)_config_env=AR="$($(package)_ar)" RANLIB="$($(package)_ranlib)" CC="$($(package)_cc)"
 $(package)_config_opts=--prefix=$(host_prefix) --openssldir=$(host_prefix)/etc/openssl
+$(package)_config_opts+=no-afalgeng
+$(package)_config_opts+=no-asm
+$(package)_config_opts+=no-async
+$(package)_config_opts+=no-autoalginit
+$(package)_config_opts+=no-autoerrinit
```

Comment:
``no-autoalginit`` and ``no-autoerrinit`` are potentially risky, because they require that the app explicitly initialize stuff that it wouldn't otherwise need to initialize, and we don't have time for the necessary review. Let's remove these for the time being.